control/controlclient/direct.go

```diff
@@ -602,6 +603,17 @@ func (c *Direct) doLogin(ctx context.Context, opt loginOpt) (mustRegen bool, new
 	if persist.NetworkLockKey.IsZero() {
 		persist.NetworkLockKey = key.NewNLPrivate()
 	}
+
+	// attempt to generate a new hardware attestion key if none exists
+	if persist.AttestationKey == nil {
```

Comment:
can we do this in `ipnlocal.LocalBackend.Start` or somewhere similar, so that existing logged-in clients can generate the attestation key without waiting for a fresh login?